pages/product-listing/product-listing.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jewelory</title>
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="./product-listing.css">
```

Comment:
Try to import all the CSS files in one style.css file & link that style.css files into head tag.